src/app/components/projects/site-cards/site-cards.component.html

```diff
@@ -1,5 +1,5 @@
 <h2>
-  Sites
+  {{ region ? "Points" : "Sites" }}
```

Comment:
I'm not sure using `region` is a good discriminator here. 

It seems like adding a `isPoint` property to the `site` model would work better.

Reply:
I didn't understand this. Revoking